### DISCUSSION TITLE
「天气app」 的通知中，不显示中国的城市

### DISCUSSION THREAD

**Discussion by Ipkarkin** [NSRingo/.github#1] [Q&A 问答] [answered]
都用qx跑weatherkit,只有mac上有国内的可以勾选,iPhone和iPad上都不生效,这个是怎么操作呢?
<img width="459" alt="截屏2024-10-13 22 02 17" src="https://github.com/user-attachments/assets/5ec024fd-fed5-420f-ae2b-0af5fce81e2d">

![IMG_9328](https://github.com/user-attachments/assets/8d14a07a-0fe9-43d8-8923-72b67181f2d1)

**Comment by VirgilClyne** [comment, marked as answer]
启用脚本前添加的城市的通知选项不会立即生效
您可以尝试：
1. 耐心等待系统下一次主动检测各个城市的功能可用性（约72小时后）
2. 在 天气app 中删除后重新添加城市
3. 卸载重装 天气app
4. 重启设备

**Reply by Banben07** [reply]
请问当前定位的位置是默认通知的吗，无法添加当前定位

**Comment by zxyblzcat** [comment]
起作用之后关闭代理还有效果吗
